packages/lesswrong/server/migrations/20231101T215739.add_givingSeasonNotifyForVoting_and_election_candidate_links.ts

```diff
@@ -31,16 +31,16 @@ export const acceptsSchemaHash = "0cc5ac04b5c4340a894f1bef511f22a9";
 
 import ElectionCandidates from "../../lib/collections/electionCandidates/collection";
 import Users from "../../lib/collections/users/collection";
-import { addField, dropField } from "./meta/utils"
+import { addField, addRemovedField, dropField, dropRemovedField } from "./meta/utils"
 
 export const up = async ({db}: MigrationContext) => {
-  await addField(db, Users, "givingSeasonNotifyForVoting");
+  await addRemovedField(db, Users, "givingSeasonNotifyForVoting");
```

Comment:
I think this kind of thing doesn't work? see `20230331T002047.add_noComicSans_to_Users.ts`

Reply:
@oetherington Can you double check this? I think we ran into issues last time we dropped a field

Reply:
Thanks - I forgot about this! Should be fixed now.